TugboatQA/images
# Refactor to task and bake; add support for multi-platform images

To test:

1. Create a .env.local
2. Add the AWS environment variables (see **Tugboat AWS S3 Access Key** in 1Password) in .env.local (see .env)
3. Set the`NAMESPACE` to your Docker hub username in .env.local
4. Run `docker login` to the account above
5. Make sure you have [`task`](https://taskfile.dev) v3.28.0 or newer installed.
6. Make sure you have GNU Parallel installed. (`brew install parallel` on macOS, `apt-get install parallel` on Debian / Ubuntu)
7. Run `PUSH=1 time task`
8. Make note of any errors you may encounter.
9. Make note of the time the command took to run.
10. Try running the command again if you encounter errors and see if it works from there.
11. Make note of the time the command took to run.
12. Run `task clean-all` to clean everything up afterward. Ensure all generated files are cleaned out, images have been removed, and `docker buildx ls` doesn't show the `tugboatqa-bakery` builder.

Resolves #104
